libs/ccf-shared/src/lib/services/data-source/api-endpoint.service.ts

```diff
@@ -101,11 +104,39 @@ function spatialSceneNodeReviver(nodes: RawSpatialSceneNode[]): SpatialSceneNode
   providedIn: 'root',
 })
 export class ApiEndpointDataSourceService implements DataSource {
+  private readonly http = inject(HttpClient);
```

Comment:
Hmm, this is ok to do for this PR, but when you replace the CCF-API client with the @hra-api/ng-client you'll want to use the equivalent function from there. I think the logic will be similar to below.